SumoLogic.Logging.Log4Net/BufferedSumoLogicAppender.cs

```diff
@@ -60,7 +60,7 @@ public class BufferedSumoLogicAppender : AppenderSkeleton
         public BufferedSumoLogicAppender()
             : this(null, null)
         {
-        }
+		}

```

Comment:
sorry I had a change in this file, and then un-did it. I guess I left the whitespace in by accident